include/z64.h

```diff
@@ -157,15 +157,21 @@ typedef struct {
     /* 0x4 */ f32 dynamicSizeStep;
     /* 0x8 */ u8 state;
     /* 0x9 */ u8 sizeGrowsCos2;
-    /* 0xA */ u8 unkA;
+    /* 0xA */ u8 colorsIndex;
     /* 0xB */ u8 flags;
-    /* 0xC */ u8 unkC;
+    /* 0xC */ u8 lightParamsIndex;
 } FireObjInitParams; // size = 0xD
 
-typedef struct {
+typedef struct FireObjColors {
     /* 0x0 */ Color_RGBA8 primColor;
-    /* 0x4 */ u8 unk4;
+    /* 0x4 */ u8 lod;
```

Comment:
What does `lod` mean? If it's an abbreviation for something, could you put what the abbreviation stands for in comments next to this?

Reply:
I assume this is the standard "level of detail" thing we see all over with primitive colour.

Reply:
Interesting. I've never heard of that term before but I also don't work a lot with colours and graphics

Reply:
Adding onto this, it does seem like `lod` is a common term and it's just my lack of knowledge. Maybe a comment `// level of detail` would be nice, but it's not necessary.

Reply:
It's used elsewhere in stuff like SkelAnime, that's how I know about it.